app/src/main/java/ch/epfl/skysync/screens/UserDetails.kt

```diff
@@ -0,0 +1,75 @@
+package ch.epfl.skysync.screens
+
+import android.annotation.SuppressLint
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowBack
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.FabPosition
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.SmallTopAppBar
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBarDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import androidx.navigation.NavHostController
+import ch.epfl.skysync.components.FlightsList
+import ch.epfl.skysync.models.flight.FinishedFlight
+import ch.epfl.skysync.models.user.User
+import ch.epfl.skysync.navigation.AdminBottomBar
+import ch.epfl.skysync.ui.theme.lightOrange
+
+@SuppressLint("UnusedMaterial3ScaffoldPaddingParameter")
+@Composable
+fun UserDetailsScreen(
+    navController: NavHostController,
+    allFlights: List<FinishedFlight>,
+    user: User
+) {
+  val userFlights =
+      allFlights.filter { flight -> flight.team.roles.any { role -> role.assignedUser == user } }
+
+  Scaffold(
+      modifier = Modifier.fillMaxSize(),
+      topBar = { UserDetailTopAppBar(user = user, onBackClicked = { navController.navigateUp() }) },
+      bottomBar = { AdminBottomBar(navController = navController) },
+      floatingActionButtonPosition = FabPosition.End,
+  ) { paddingValues ->
+    FlightsList(userFlights, lightOrange, paddingValues, "Completed Flights") { selectedFlight ->

```

Comment:
maybe take a different color for the title banner to better distinguish the home screen (upcoming flights) from the User details screen

Reply:
Yes we could but I thought that it was nice to keep color coherence through the app especially for screen that are so far away